augur/cli/db.py

```diff
@@ -16,6 +16,7 @@
 from sqlalchemy import exc
 
 from augur.cli import pass_config, pass_application
+from augur.platform_connector import PlatformConnector
```

Comment:
This adds features for collecting new repos added to an organization.  It is not a duplicate of existing features or functions.